tables-test-fixtures/tables-test-fixtures-iceberg-1.2/src/main/java/com/linkedin/openhouse/tablestest/OpenHouseSparkITest.java

```diff
@@ -23,7 +23,7 @@ public class OpenHouseSparkITest {
    *
    * <p>Do not {@link SparkSession#close()} a created SparkSession.
    */
-  protected synchronized SparkSession getSparkSession() throws Exception {
+  protected static synchronized SparkSession getSparkSession() throws Exception {
```

Comment:
why and what's the implication of this change?

Reply:
removed this code